app/views/attribute.py

```diff
@@ -14,6 +18,65 @@ class AttributeViewSet(viewsets.ModelViewSet):
     filter_backends = [filters.DjangoFilterBackend]
     filterset_fields = ["course_id"]
 
+    def save_attribute_option(
+        self, attribute_option_data: dict, attribute_id: int
+    ) -> AttributeOption:
+        attribute_option_serializer = AttributeOptionSerializer(
+            data={**attribute_option_data, "attribute": attribute_id}
+        )
+        if not attribute_option_serializer.is_valid():
+            raise ValidationError(attribute_option_serializer.errors)
+        attribute = get_object_or_404(Attribute, pk=attribute_id)
+
+        attribute_option_id = attribute_option_data.get("id")
+        if attribute_option_id:
+            attribute_option = get_object_or_404(
+                AttributeOption, pk=attribute_option_id
+            )
+            AttributeOption.objects.filter(pk=attribute_option_id).update(
```

Comment:
```suggestion
            attribute_option.update(
```

Don't you already have the object instance that you want to update?

Reply:
https://stackoverflow.com/questions/67250135/why-is-it-not-possible-to-update-a-django-model-instance-like-this-instanceu
I believe there's no update function for django object instance unfortunately so we cannot do .update on attribute_option